javascript/app-framework/src/components/DataShaperApp/DataShaperApp.types.ts

```diff
@@ -17,4 +34,14 @@ export interface DataShaperAppProps {
 	 * TODO: handle this internally
 	 */
 	children: React.ReactNode
+
+	/**
+	 * The currently selected route
+	 */
+	selectedRoute?: string
```

Comment:
Suggest naming as `selectedKey` to align with common control convention. Consumer doesn't necessarily know it is a "route" anyway